include/nuttx/net/ioctl.h

```diff
@@ -125,7 +125,7 @@
 
 /* Network socket control ***************************************************/
 
-#define SIOCDENYINETSOCK _SIOC(0x0033) /* Deny network socket. */
+#define SIOCDENYINETSOCK _SIOC(0x0036) /* Deny network socket. */
```

Comment:
duplicate with line 132:
`#define SIOCBRADDBR        _SIOC(0x0036)  /* Create new bridge device */`

Reply:
@anchao san
Thank you for your comment.
I change it now.